src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/UserControllerGetUsersIntTest.java

```diff
@@ -56,14 +72,74 @@ void usersGetShouldReturnForbiddenError() throws Exception {
     }
 
     @Test
-    void usersGetShouldReturnNotImplemented() throws Exception {
-        adminUserStub.givenUserIsAuthorised(mockUserIdentity);
-
-        mockMvc.perform(get(ENDPOINT_URL).queryParam("courthouse", "-1"))
-            .andExpect(status().isNotImplemented());
+    void getUsersAuthorised() throws Exception {
+        UserAccountEntity user = adminUserStub.givenUserIsAuthorised(mockUserIdentity);
+
+        createEnabledUserAccountEntity(user);
+
+        mockMvc.perform(get(ENDPOINT_URL)
+                            .header("Content-Type", "application/json")
+                            .header(EMAIL_ADDRESS, "james.smith@hmcts.net")
+                            .queryParam(COURTHOUSE_ID, "21"))
```

Comment:
This query param isn't used so could be removed, but I guess it's not doing any harm.

Reply:
queryParam removed